packages/flutter/lib/src/rendering/semantics.dart

```diff
@@ -46,6 +46,25 @@ typedef void SemanticsAnnotator(SemanticsNode semantics);
 /// Used by [SemanticsNode.visitChildren].
 typedef bool SemanticsNodeVisitor(SemanticsNode node);
 
+/// A tag for a [SemanticsNode].
+///
+/// Tags can be interpreted by the parent of a [SemanticsNode]
+/// and depending on the presence of a tag the parent can for example decide
+/// how to add the tagged note as a child.
+///
+/// As an example, the [RenderSemanticsGestureHandler] uses tags to determine
+/// if a child node should be excluded from the scrollable area for semantic
+/// purposes.
+class SemanticsTag {
+  const SemanticsTag(this.name);
```

Comment:
this constructor needs docs, probably something that talks about how the argument is ignored for the purposes of identity.